elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/Schema.java

```diff
@@ -281,4 +292,18 @@ protected Dimension constructDimension(String dimensionField, Class<?> cls, Enti
                         )
                 );
     }
+
+    /**
+     * Returns whether or not an entity field is a toMany relationship.
+     *
+     * @param dimensionField  The entity field to check
+     * @param cls  The entity that contains the field
+     *
+     * @return {@code true} if the field is toMany
+     */
+    private boolean isToManyRelationship(String dimensionField, Class<?> cls) {
```

Comment:
There already is a method in the dictionary which does this:

getRelationshipType